licensedb/dataset_test.go

```diff
@@ -6,6 +6,8 @@ import (
 	"sync"
 	"testing"
 
+	"gopkg.in/src-d/go-license-detector.v2/licensedb/api"
```

Comment:
Can you please move this import below the existing `"github.com/stretchr/testify/assert"`